onnxruntime/core/providers/dml/OperatorAuthorHelper/MLOperatorAuthorHelper.h

```diff
@@ -12,6 +12,8 @@
 #define ML_CHECK_BOOL(x) THROW_HR_IF(E_INVALIDARG, !(x))
 #endif
 
+#include <optional>
```

Comment:
We typically include standard headers in the precomp.h, not in each header. For contrast, `std::vector` is also used in this file, but there is no `#include <vector>` here because it's in the precomp.h:

```
#include <limits>
#include <cassert>
#include <chrono>
#include <vector>
#include <map>
#include <set>
#include <numeric>
```

Reply:
@jeffbloo Note, CR still open.

Reply:
@jeffbloo Note, CR still open.

Reply:
@jeffbloo Note, CR still open. Abandon it?